### ISSUE TITLE
Cannot get autoupdate working on formula

### ISSUE THREAD

**Issue by svenschuldt** [mdelobelle/metadatamenu#545]
**What I’m trying to do**

In the definition of a class I have defined three fields, two of them being numbers, one of them being a formula - looks as follows:

![image](https://github.com/mdelobelle/metadatamenu/assets/154823555/701782a8-9860-4497-9c8c-4a0dbce91aa9)

the field totalCalc is simply defined:

![image](https://github.com/mdelobelle/metadatamenu/assets/154823555/576f4f99-b8ea-4464-8bf5-717d8f45dd66)

I want the field `totalCalcbeing` automatically updated based on `current.calc1 + current.calc2`.

However, I cannot get the formula field totalCalcto autoupdate - whether in the frontmatter of a note:

![image](https://github.com/mdelobelle/metadatamenu/assets/154823555/7c3345dd-c982-4f38-bb48-f80f4864f1d7)

nor in the body of a note:

![image](https://github.com/mdelobelle/metadatamenu/assets/154823555/32162b9b-fe7d-4ba3-8e3a-1fb463216265)

Also, when I view the table-view of the class definition file, I cannot see any update of the `totalCalc` field:

**Things I have tried**

- read the metadata Menu Git pages
- watched the related demo videos
- searched the Obsidian forum
- searched this forum
- searched internet
- tried different examples

Any advice would be great, thanks!

**Comment by svenschuldt** [comment]
I have meanwhile done some testing:
- it must have to do with the current vault - I generated a new one and it worked
- Step by step I have added/activated all css snippets, templates, and plugins into the new vault, which are in the existing one
- after every change I made a test with metadata menu and a formula field 

The new vault works completely with formula in metadata menu, with all plugins, templates and other additions. 

So it must be the existing vaults, but have added all plugins., snippets, and templates - I am helpless.

**Comment by Felharrim** [comment]
Is auto-calculation enabled in the plugin settings?

Yesterday I was figuring out this plugin myself and came across one interesting point. After creating the formula, the field with it was not added to the metadata field until I entered this property in the inline form sun::

**Comment by svenschuldt** [comment]
I have read your sentence "until I entered this property in the inline field” several times, but I am not getting my head around this, what you mean. 

A comment I forgot to make: the formula works when it's field is not categorized as an autoupdate field. 

And yes: in the settings, I have the Auto Calculation turned on ...

**Comment by Felharrim** [comment]
I didn't express myself quite correctly. I wanted to say that the property is not added to the field for metadata until I add the same property to the field of the note itself. Example(Sum:: 5)

**Comment by svenschuldt** [comment]
I see, yes, that is correct — but still, the auto-update does not work. I'm adding the field to the note itself, and it does not update — neither for a first calculation nor if any change is made to the “source-fields”.

**Comment by Felharrim** [comment]
I understood your comment well. A field with a formula starts working when auto-update is disabled in its settings, i.e. quite the opposite?

Do the plugin versions in the working voult and the test one match?

**Comment by svenschuldt** [comment]
Yes, the plugins are the same version - I watched out for versions for every plugin I installed. 
It must be somewhere in the vault, but I don't know where to look.

**Comment by Felharrim** [comment]
If you still find a working option, then do everything the same as you did, adding other plugins one by one and check it on the test storage

**Comment by mdelobelle** [comment]
Does the option 
<img width="578" alt="image" src="https://github.com/mdelobelle/metadatamenu/assets/1233240/cce0afb0-0bb7-429a-b50b-98f72328430c">
from the command palette make the formula updating properly?
